remix_01.py

```diff
@@ -39,11 +39,18 @@ def filter_json(id):
     user_name = data['user_name']
     bpm = data['upload_extra']['bpm']
     upload_name = data['upload_name']
+    remix_parents = []
     if data.has_key('remix_parents'):
-        remix_parents = data['remix_parents'][0]['upload_id']
-    else:
-        remix_parents = "---"
-    remix_children = data['remix_children'][0]['upload_id']
+        for parent in data['remix_parents']:
+            remix_parents.append(parent['upload_id'])
+    remix_children = []
+    if data.has_key('remix_children'):
+        for child in data['remix_children']:
```

Comment:
We do the same to create a list of children. There is just one extra problem with some children don't have an `upload_id`, instead they have a `pool_item_id` ... (not sure if that is the id that we would like)